modules/qna/src/views/full/Components/ContentAnswer/index.tsx

```diff
@@ -0,0 +1,59 @@
+import cx from 'classnames'
+import React, { FC, Fragment } from 'react'
+
+import style from './style.scss'
+
+interface Props {
+  // TODO change once typings are implemented
+  content: any
+  active: boolean
+  onEdit: () => void
+}
+
+const ContentAnswer: FC<Props> = ({ content, onEdit, active }) => {
+  const renderCardOrImg = ({ image, title }): JSX.Element => {
+    return (
+      <Fragment>
+        {image && <div style={{ backgroundImage: `url('${image}')` }} className={style.img}></div>}
+        {title}
+      </Fragment>
+    )
+  }
+
+  const ellipsisText = (text: string): string => {
+    if (text.length > 51) {
+      text = `${text.substr(0, 51)}${text.substr(-1) !== '.' ? '...' : ''}`
```

Comment:
This checks if the last char of the whole text is a dot instead of the cut part.